doc/spec/support-assessment.md

```diff
@@ -0,0 +1,237 @@
+  maintain consistency. (The alternative would be multiple instances
+  with varying staleness)
+- Simplified configuration
+
+### Dashboard
+
+The dashboard part is out of scope for this proposal. Initially we have
+a simple JSON HTTP endpoint exporting support assessment data.
+
+### Custom Exporter
+
+We will add or extend existing Prometheus exporters where necessary.
+
+Where we create new exporters we may want to consider adding
+additional `/detail/$metric_name`-style endpoints. Bubbles could query
+them to receive additional information for human consumption not
+available from Prometheus directly.
+
+## Alternatives
+
+### Cephadm Configchecks
+
+Configuration checks build into the cephadm manager module. Tightly
+integrated with the module for rule execution and command handling.
+Creates an in memory copy of relevant cluster state. Has no query or
+rule engine abstraction on top of the gathered data. Checks are
+Python code processing the gathered data directly. Failing
+checks raise Ceph health warnings. Manager commands can enable and
+disable individual checks. Check code is part of the checker.
+
+Supported checks:
+- Consistent kernel security modules
+- Host subscription state
+- Hosts connected to the Ceph cluster public network
+- Common MTUs and link speeds
+- Hosts have the cluster and public networks configured
+- Consistent kernel and Ceph versions
+
+### Build Our Own Gathering and Rule Execution Framework
+
+Similar to how configchecks lives in cephadm build a data gathering and
+rule execution framework in a Bubbles module.
+
+Rule execution might be done with rational queries or
+even unification. There are Python rule engine libraries, but
+possibly nothing as battle tested as Prometheus. A MiniKanren
+implementation and Durable Rules was briefly evaluated and considered
+too risky.
+
+Data gathering is also challenging: Sources may change formats,
+joining is not always straight forward because of different
+naming schemes. Keeping the memory footprint low and supporting
+historical queries is challenging.
+
+## Why Prometheus?
+
+Prometheus a de facto industry standard time series database and
+alerting system. It is stable and has good performance. We can
+leverage [many](https://prometheus.io/docs/instrumenting/exporters/) existing exporters. PromQL is widely understood and
+allows interactive rule development. Custom exporters are easy to
+write.
+
+There are synergetic effects with Ceph, especially when adding more
+and more interesting metrics.
+
+## Consequences
+
+Going with Prometheus saves us from building our own rule engine or
+writing a lot of code to process state into alert conditions
+(configchecks.py). Writing rules will become easier the more we have
+and the more data sources we integrate. We will loose some flexibility, as
+we can't "just" query arbitrary string state data without processing
+it first.
+
+## Status
+
+Proposed. Prototype demoed.
```

Comment:
Let's add a link here and make it public. @jecluis did you save the recordings or was that @ml8mr ?